README.md

```diff
@@ -1,23 +1,150 @@
-# README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+* Ruby version<a name="readme-top"></a>
 
-Things you may want to cover:
+<div align="center">
 
-* Ruby version
+  <h3><b>Hello World</b></h3>
 
-* System dependencies
+</div>
 
-* Configuration
+<!-- TABLE OF CONTENTS -->
 
-* Database creation
+# 📗 Table of Contents
 
-* Database initialization
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Hellow-World\] ](#-hellow-world-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
-* How to run the test suite
+# 📖 [Hellow-World] <a name="about-project"></a>
+This is hello world  application
+> .
 
-* Services (job queues, cache servers, search engines, etc.)
 
-* Deployment instructions
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><p> <b>Ruby: </b> The primary programming language for the project.</p></li>
+    <li><p><b>Rails: </b> It is a ruby frameworks.</p></li>
+    <li><p><b>React: </b> It is a javacsript frameworks.</p></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>none</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>Postgresql</li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+</ul>
+
+<h1>🚀 Live Demo</h1>
+
```

Comment:
- [ ] Since you have no live demo link, then kindly remove this section and the link on the header.